### DISCUSSION TITLE
YTVanced by Cyunu

### DISCUSSION THREAD

**Discussion by Kel-Morian** [ReVanced/revanced-discussions#775] [General] [answered]
Hello, I would have liked to know what is the difference with the YTVanced by Cuynu?
I use the latter and wanting to update the apk I realized the existence of Revanced.
Is it worth migrating to this one?

**Comment by oSumAtrIX** [comment, marked as answer]
From skimming through it, it is an unofficial and closed source distributor of ReVanced.

**Reply by Kel-Morian** [reply]
Ok thank you.
I'm looking to install Revanced through Revanced Manager.
It's still much more complicated than Cuynu's method in the sense that it offers a separate apk from the original Youtube application.
Basically it boils down to two APKs, YTVanced and MicroG, we install and it works directly.
It took me a few seconds.

I just finished installing Revanced and it's pretty heavy casually. I had to uninstall all Youtube updates, get an APK of the correct version of Youtube that Vanced Manager claims (without being sure that the APK is safe), repatch the application again (and that was relatively long), then install Revanced.

And it's silly but currently, by testing Revanced, I can't find any added value?
Or am I wrong somewhere and would like someone to explain it to me?
Unless Cuynu just took over the work of Revanced who was there before and did all the work?

And another question, why not use the latest version of MicroG offered on the official MicroG Github?

**Reply by oSumAtrIX** [reply]
> without being sure that the APK is safe

You were installing an APK without being sure it is safe to begin with. You are asked to source your own APK to patch, it is something out of our responsibility.

Cuynu is doing the same process to patch with ReVanced. 
ReVanced on the other hand will not share any prepatched APK as it is not legal, which is why you are required to patch it yourself. 
The ReVanced Manager is still alpha.
